salt/_modules/metalk8s_volumes.py

```diff
@@ -295,7 +296,7 @@ def prepare(self, force=False):
                             .format(self.path))
         storage_class = self.get('spec.storageClass')
         # If we got a string that means the name wasn't replaced by the object.
-        if isinstance(storage_class, basestring):
+        if isinstance(storage_class, six.string_types):
```

Comment:
What about l.222-223 of this file? :stuck_out_tongue: 

You'll probably want https://stackoverflow.com/a/35673504

Reply:
Oh ! Right on my py3 test didn't faced any issue about this one. Anyway I will change this one